### DISCUSSION TITLE
Stable Release v2.1.1 (React Update)

### DISCUSSION THREAD

**Discussion by TrainDoctor** [SteamDeckHomebrew/decky-loader#192] [Announcements]
I want to congratulate everyone that participated in this months long effort and that got us this far. We couldn't have made it this far without the community contribution and we have yet more planned. If you're looking for a feature to add to decky-loader or one of it's subsidiary libraries or pieces of software, please check the tracker [here](https://github.com/orgs/SteamDeckHomebrew/projects/9). We try to keep it up to date as best as possible. Thanks for your patience, and here's to the future.

## What's Changed
* react: Add Router hook & fix typescript issues by @AAGaming00 in https://github.com/SteamDeckHomebrew/decky-loader/pull/68
* Add contributor install script by @TrainDoctor in https://github.com/SteamDeckHomebrew/decky-loader/pull/69
* Implement React-based plugin store by @AAGaming00 in https://github.com/SteamDeckHomebrew/decky-loader/pull/81
* remove body property in args by @hulkrelax in https://github.com/SteamDeckHomebrew/decky-loader/pull/91
* Uninstall functionality by @botatooo in https://github.com/SteamDeckHomebrew/decky-loader/pull/97
* Use deckyState in uninstall menu (fixes #98) by @botatooo in https://github.com/SteamDeckHomebrew/decky-loader/pull/100
* Fixed plugin installation ssl verification issue by @WerWolv in https://github.com/SteamDeckHomebrew/decky-loader/pull/101
* Use call instead of Popen to get rid of zombie processes by @botatooo in https://github.com/SteamDeckHomebrew/decky-loader/pull/113
* CI revamp by @botatooo in https://github.com/SteamDeckHomebrew/decky-loader/pull/110
* change ci again by @botatooo in https://github.com/SteamDeckHomebrew/decky-loader/pull/116
* Updater for decky-loader by @AAGaming00 in https://github.com/SteamDeckHomebrew/decky-loader/pull/117
* Fix jq errors in prerelease script by @bycEEE in https://github.com/SteamDeckHomebrew/decky-loader/pull/118
* Readme.md references incorrect github repo by @beebls in https://github.com/SteamDeckHomebrew/decky-loader/pull/124
* Also change the Uninstall script to the new decky-loader repo name by @beebls in https://github.com/SteamDeckHomebrew/decky-loader/pull/125
* Use Environment Variables by @pastaq in https://github.com/SteamDeckHomebrew/decky-loader/pull/123
* Don't pass unzip job to event loop by @Scrumplex in https://github.com/SteamDeckHomebrew/decky-loader/pull/136
* Remove console.log that was causing lots of log spam by @OMGDuke in https://github.com/SteamDeckHomebrew/decky-loader/pull/138
* Add Plugin Reload Button to Settings by @Scrumplex in https://github.com/SteamDeckHomebrew/decky-loader/pull/128
* Add CEF Remote Debugging toggle by @Scrumplex in https://github.com/SteamDeckHomebrew/decky-loader/pull/129
* Better wrapping of plugin tags by @ToyVo in https://github.com/SteamDeckHomebrew/decky-loader/pull/150
* set gid along with uid by @simons-public in https://github.com/SteamDeckHomebrew/decky-loader/pull/152
* Refractor plugin backend by @botatooo in https://github.com/SteamDeckHomebrew/decky-loader/pull/111
* Fix LimitOverrunError when a lot of data is passed to the backend (fixes #158) by @popsUlfr in https://github.com/SteamDeckHomebrew/decky-loader/pull/161
* Addition of proper branch slection by @TrainDoctor in https://github.com/SteamDeckHomebrew/decky-loader/pull/168
* Download Remote Binaries during Store Install by @joamjoamjoam in https://github.com/SteamDeckHomebrew/decky-loader/pull/177
* Grab Folder Name instead of .zip name by @joamjoamjoam in https://github.com/SteamDeckHomebrew/decky-loader/pull/178
* Add note about opening terminal to README by @saamirt in https://github.com/SteamDeckHomebrew/decky-loader/pull/176
* updated installation steps by @DanBurkhardt in https://github.com/SteamDeckHomebrew/decky-loader/pull/180
* (readme) Make store link go to beta.deckbrew + Change wording by @PartyWumpus in https://github.com/SteamDeckHomebrew/decky-loader/pull/184
* fix(pluginview): align icon and plugin name by @FrogTheFrog in https://github.com/SteamDeckHomebrew/decky-loader/pull/185
* Change 'PluginLoader' to 'Decky Loader' in readme by @PartyWumpus in https://github.com/SteamDeckHomebrew/decky-loader/pull/190

## New Contributors
* @AAGaming00 made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/68
* @hulkrelax made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/91
* @botatooo made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/97
* @WerWolv made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/101
* @bycEEE made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/118
* @beebls made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/124
* @pastaq made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/123
* @Scrumplex made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/136
* @OMGDuke made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/138
* @ToyVo made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/150
* @simons-public made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/152
* @popsUlfr made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/161
* @joamjoamjoam made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/177
* @saamirt made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/176
* @DanBurkhardt made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/180
* @PartyWumpus made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/184
* @FrogTheFrog made their first contribution in https://github.com/SteamDeckHomebrew/decky-loader/pull/185

**Full Changelog**: https://github.com/SteamDeckHomebrew/decky-loader/compare/v1.3.1...v2.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/SteamDeckHomebrew/decky-loader/releases/tag/v2.1.1'>Stable Release v2.1.1 (React Update)</a>.</em>

**Comment by adntaha** [comment]
Wow! It's definitely been a wild ride coming this far.

**Comment by beebls** [comment]
I feel weird being included among all of these huge contributions when all I did was edit a readme 😳

Amazing people working on this and so glad to be a part of the ecosystem

**Reply by TrainDoctor** [reply]
> I feel weird being included among all of these huge contributions when all I did was edit a readme flushed
> 
> Amazing people working on this and so glad to be a part of the ecosystem

"A contributor is a contributor, no matter how small."

**Comment by infernn** [comment]
How i can update It?
